integration_test.go

```diff
@@ -0,0 +1,59 @@
+
+func TestIntegration(t *testing.T) {
+	srv := httptest.NewServer(
```

Comment:
this is an actual FakeServer (to avoid the dependency on Jira being up to run the tests. We'll execute queries against this server instead....